src/nti/contentfragments/configure.zcml

```diff
@@ -67,6 +67,10 @@
              for=".interfaces.IUnicode"
              provides=".interfaces.IHTMLContentFragment" />
 
+    <adapter factory=".interfaces.RstContentFragment"
```

Comment:
Hmm. The `TODO` from directly above applies as much so here, I think, and probably even more. While most HTML parsers just ignore junk and keep on going, docutils doesn't. Errors in ReST markup can kill the rendering process. 

It seems to me that perhaps this should be something like `factory=".rst.check_user_rst"`, where that function calls into docutils to do some basic verification? 

`check` not `sanitize` because I'm not sure that it is possible to sanitize ReST directly? However, the code in https://github.com/pypa/readme_renderer/blob/master/readme_renderer/rst.py (as used on PyPI to handle arbitrary `long_description` uploads) is probably of interest.

Reply:
If we're embedding this for direct display to the user in arbitrary contexts, as https://github.com/NextThought/nti.assessment/pull/11/ suggests, then I do think this needs to go through validation with similar settings as `readme_renderer` uses. Notably, allowing embedded arbitrary HTML is potentially dangerous.